src/interpolations.jl

```diff
@@ -1973,4 +1966,36 @@ adjust_dofs_during_distribution(::Nedelec{RefTetrahedron, 1}) = false
 function get_direction(::Nedelec{RefTetrahedron, 1}, j, cell)
     edge = edges(cell)[j]
     return ifelse(edge[2] > edge[1], 1, -1)
-end
\ No newline at end of file
+end
+
+
+# RefHexahedron, 1st order Lagrange
+# https://defelement.org/elements/examples/hexahedron-nedelec1-lagrange-1.html
+function reference_shape_value(ip::Nedelec{RefHexahedron, 1}, ξ::Vec{3}, i::Int)
+    x, y, z = ξ
+    nil = zero(x)
+
+    i == 1 && return Vec(0.125 * (y * z - y - z + 1), nil, nil)
```

Comment:
Ensures that it doesn't convert to `Float64` for `Float32` input. 
```suggestion
    i == 1 && return Vec((y * z - y - z + 1)/8, nil, nil)
```